builder/vsphere/common/step_boot_command.go

```diff
@@ -87,11 +87,16 @@ func (s *StepBootCommand) Run(ctx context.Context, state multistep.StateBag) mul
 			keyShift = down
 		}
 
+		shift := down
+		if keyShift {
```

Comment:
Will keyshift ever be anything other than `down`?

Reply:
`keyshift` can be false when `down` is true and we need to use the value of `down` in this case because it means that the driver knows that the char is shifted and the user didn't write `<leftShiftOn>|<leftShiftOff>`  for that. But if the user set `<leftShiftOn>|<leftShiftOff>`  for itself we need to make sure to keep it in the `keyshift` that’s why if that is true we use it because it means the shift is pressed by the user.